### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Site_Name link_IP Restrictions: The link “here” is not descriptive and meaningful.

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#194]
**Environment Details:**
URL: https://manage.iis.net/
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**WCAG ID Violated: WCAG2.4.4**

GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #C+E-H; #.NETWebsites; #Edge; #Chrome; #Narrator; #NVDA; #WCAG2.4.4

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Site and press enter on any one of the name link field
5.	Tab till IP Restrictions and press enter
6.	Validate whether all the links are descriptive and meaningful

**Actual Result:**
The link “here” is not descriptive and meaningful

**Expected Result:**
The link “here” should be descriptive and meaningful

**User Impact:**
Users with disabilities finds it difficult in understanding the functionality of links if they are not descriptive

**Attaching screenshots for your reference:**
![bug 194_links](https://user-images.githubusercontent.com/42638744/44846646-8dc9e900-ac6e-11e8-804c-0fec90371662.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#Closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.iis.net%2F&data=05%7C01%7Cv-bhaprakash%40microsoft.com%7C7a737faa4af14c4ad31a08db634ab100%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638212945750407456%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Q7lvppcGrDLIRxk6EaLiumHk2i6E5S784T%2BniobLEu4%3D&reserved=0) home page will be removed.
